app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -437,6 +437,9 @@ public void onMenuBrowsingItemClicked(View v) {
                 TelemetryWrapper.clickToolbarShare();
                 break;
             case R.id.capture_page:
+                if(Settings.getInstance(this).shouldShowScreenshotOnBoarding()) {
```

Comment:
I think simply doing Settings.getInstance(this).setScreenshotOnBoardingDone(); since another branch (jump) may be actually more costly then writing to preference.